test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -75,7 +75,7 @@ describe('33acrossBidAdapter:', function () {
       return this;
     };
 
-    this.withViewabiliuty = viewability => {
```

Comment:
Fix typo from earlier releases. Does not have anything to do with current feature